### DISCUSSION TITLE
Timed Detonator UI desync

### DISCUSSION THREAD

**Discussion by limpopo1400** [FakeFishGames/Barotrauma#14964] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When I select the delay of the explosion in the new timed detonator, it just selects a random number in the input field, and this also happens with a check mark.

### Reproduction steps

1. Create p2p lobby or join to already hosted lobby
2. spawn timed detonator and open UI

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
